dart/dynamics/ReferentialSkeleton.hpp

```diff
@@ -70,6 +80,15 @@ class ReferentialSkeleton : public MetaSkeleton
   /// \{ \name Structural Properties
   //----------------------------------------------------------------------------
 
+  /// Returns number of skeletons associated with this ReferentialSkeleton.
+  std::size_t getNumSkeletons() const;
+
+  /// Returns skeletons associated with this ReferentialSkeleton.
+  const std::unordered_set<const Skeleton*>& getSkeletons() const;
+
+  /// Returns whether this ReferentialSkeleton contains \c skel.
```

Comment:
Ambiguous comment. This could be interpreted as "contains the _entire_ `Skeleton`" or "contains _any_ `BodyNode` or `Joint` from the `Skeleton`." I think we intend the latter.

Reply:
Done.